docs/tasks/debug-application-cluster/audit.md

```diff
@@ -208,153 +127,51 @@ rules:
 The [audit profile used by GCE][gce-audit-profile] should be used as reference by
 admins constructing their own audit profiles.
 
-### Audit backends
-
-Audit backends implement strategies for emitting events. The [kube-apiserver][kube-apiserver]
-provides a logging and webhook backend.
-
-Each request to the API server can generate multiple events, one when the request is received,
-another when the response is sent, and additional events for long running requests (such as
-watches). The ID of events will be the same if they were generated from the same request.
-
-The event format is defined by the `audit.k8s.io` API group. The `v1alpha1` format of this
-API can be found [here][audit-api] with more details about the exact fields captured.
-
-#### Log backend
+## Audit backends
 
-The behavior of the `--audit-log-path` flag changes when enabling the `AdvancedAuditing`
-feature flag. All generated events defined by `--audit-policy-file` are recorded in structured
-json format:
+Audit backends implement exporting audit events to an external storage.
+[Kube-apiserver][kube-apiserver] out of the box provides two backends:
 
-```
-{"kind":"Event","apiVersion":"audit.k8s.io/v1beta1","metadata":{"creationTimestamp":null},"level":"Metadata","timestamp":"2017-09-05T10:04:55Z","auditID":"77e58433-d345-40ac-b2d8-9866bd355cea","stage":"RequestReceived","requestURI":"/apis/rbac.authorization.k8s.io/v1/namespaces/default/roles","verb":"list","user":{"username":"kubecfg","groups":["system:masters","system:authenticated"]},"sourceIPs":["172.16.116.128"],"objectRef":{"resource":"roles","namespace":"default","apiGroup":"rbac.authorization.k8s.io","apiVersion":"v1"}}
-{"kind":"Event","apiVersion":"audit.k8s.io/v1beta1","metadata":{"creationTimestamp":null},"level":"Metadata","timestamp":"2017-09-05T10:04:55Z","auditID":"77e58433-d345-40ac-b2d8-9866bd355cea","stage":"ResponseComplete","requestURI":"/apis/rbac.authorization.k8s.io/v1/namespaces/default/roles","verb":"list","user":{"username":"kubecfg","groups":["system:masters","system:authenticated"]},"sourceIPs":["172.16.116.128"],"objectRef":{"resource":"roles","namespace":"default","apiGroup":"rbac.authorization.k8s.io","apiVersion":"v1"},"responseStatus":{"metadata":{},"code":200}}
```

Comment:
It looks like you cut out all the examples of actual audit lines? I think it would be good to retain an example, and maybe even document the API.

Reply:
I'm not sure this particular example is useful. It duplicates a lot of info in a poorer format: user can look into its own logs or into the API definition for a much better picture. Additionally, it looks like a big human-unparsable blob, so I'm against keeping it in the current form anyway.

If you want to describe the API in this docs, that's going to encounter a problem of docs obsoletion. Having better docs in the API code seems better to me, it creates a single place where it's described.

Reply:
This example does not have `RequestReceivedTimestamp` and `StageTimestamp` introduced [here](https://github.com/kubernetes/kubernetes/pull/52981)